src/trading/consts.ts

```diff
@@ -10,6 +10,7 @@ log.enabled = false
 export const DEFAULT_QUOTE_VALIDITY = 60 * 30 // 30 min
 
 export const DEFAULT_SLIPPAGE_BPS = 50 // 0.5%
+export const ETH_FLOW_DEFAULT_SLIPPAGE_BPS = 200 // 2%
```

Comment:
This is the default... for mainnet.
Other chains default to 0.5%.
I hate the duplication but I feel that's needed here as well.

Reply:
agree. If we dont want duplication, we should export the defaults too.

In the future, we should dogfood, but for now is enough to import in CoW Swap the default BPS per network

Reply:
Thank you @alfetopito !
Fixed: https://github.com/cowprotocol/cow-sdk/pull/243/commits/8a8ee01d34a66583f8a89366badea7ac1331acba